pcdsdevices/sequencer.py

```diff
@@ -0,0 +1,169 @@
+import time
+import logging
+from functools import partial
+
+from ophyd import Device, EpicsSignal, EpicsSignalRO, Component as Cpt
+from ophyd.status import DeviceStatus, SubscriptionStatus
+from ophyd.utils.epics_pvs import raise_if_disconnected
+
+
+logger = logging.getLogger(__name__)
+
+
+class EventSequencer(Device):
```

Comment:
Maybe also subclass `ophyd.flyers.FlyerInterface` in case we want to do `isinstance(obj, FlyerInterface)`

Reply:
Thank you for making me look at this file closely. It looks like I reimplemented part of `MonitorFlyerMixin` I will subclass this and `FlyerInterface`

Reply:
I didn't even notice `MonitorFlyerMixin`. Good catch.